src/apprt/gtk/flatpak.zig

```diff
@@ -0,0 +1,23 @@
+const std = @import("std");
+const Allocator = std.mem.Allocator;
+const build_config = @import("../../build_config.zig");
+const internal_os = @import("../../os/main.zig");
+const glib = @import("glib");
+
+pub fn resourcesDir(alloc: Allocator, hostAccessible: bool) !?[]const u8 {
```

Comment:
Non-snake_case variables can be confused for functions

```suggestion
pub fn resourcesDir(alloc: Allocator, host_accessible: bool) !?[]const u8 {
```